unit-tests/test.cpp

```diff
@@ -13,5 +13,16 @@ INITIALIZE_EASYLOGGINGPP
 #endif
 
 namespace test {
-    std::string context;
+
+
+std::string context;
+bool debug;
```

Comment:
No need to initialize it? where is the assignment of this variable?

Reply:
You are correct, fixed